lua/snacks/dashboard.lua

```diff
@@ -1084,6 +1084,12 @@ function M.setup()
     return
   end
 
+  -- don't open dashboard if Neovim was invoked for example `nvim +'Octo issue edit 1'`
+  if vim.fn.argc(-1) == 0 and vim.api.nvim_buf_get_name(0) ~= "" then
```

Comment:
can you remove the argc check here? That's already handled.
Just checking bufname is enough

Reply:
Done. Just out of curiosity where is this handled? Because I only see `argc(-1) > 0` in the lines above. Just so I can understand better if you don't mind.

Reply:
I see. Thanks for clarifying :smile: